ironfish/src/assert.ts

```diff
@@ -39,6 +39,12 @@ export class Assert {
     }
   }
 
+  static isGreaterThan(a: number, b: number, message?: string): void {
+    if (a <= b) {
```

Comment:
this also fails if it's equal to but the method name doesn't match that. Also the method checks if something is greater than something else but there is no greater than used in the function. The logic is inverted.